### ISSUE TITLE
Can't implement hamburger menu

### ISSUE THREAD

**Issue by liu00635** [liu00635/mtm6201-midterm#1]
@abhaysharma Hello professor, I have 3 quick questions about the midterm project:

1. I can't implement the hamburger menu on the mobile phone. It requires Javascript, so I just added the menu directly below. Is this okay？
2. The same problem occurs on the mobile search bar. I want to click the magnifying glass to pop up the search box, which also requires Javascript. Currently, the magnifying glass icon is not clickable. Is this acceptable?
3. Should I put the css for multiple pages in the same css file or separately?

Thanks,
Yali

**Comment by abhaysharma** [comment]
@liu00635 yes, you may replace the hamburger menu and search bar on mobile. Alternatively, you can look at a CSS-only hamburger menu example such as this: https://codepen.io/chr1s1202/pen/RwpQBv

It is better to use the same CSS file for all pages. If you want separate style sheets for pages, you should add one main.css to all that contains basic styles for your website and then add a second specific CSS on each page.

**Comment by liu00635** [comment]
Hello Professor,

Thank you so much!

Only one more question: When I use wave to test, it prompts that h1 has a contract issue, but it looks fine to the naked eye. Is it OK?

Sincerely,
Yali

[cid:49f0add3-760d-4377-ab32-3d972b768bf0]
________________________________

**Comment by abhaysharma** [comment]
@liu00635 Yes, the contrast is okay. 
For the future, check contrast while designing, so it does not need to be changed in development.

**Comment by liu00635** [comment]
[like]  Yali Liu reacted to your message:
________________________________
